app/views/documents/_secondary_navigation.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= render "components/secondary_navigation", {
+  aria_label: "Document",
+  items: [
+    {
+      label: t("documents.secondary_navigation.summary"),
+      href: document_path,
+      current: current_page?(document_path),
+      data_attributes: {
+        gtm: "summary-tab"
```

Comment:
We should see about changing these gtm attributes to reflect that these are no longer tabs. Maybe summary-nav and history-nav ?